yt/utilities/command_line.py

```diff
@@ -1617,65 +1634,144 @@ def __call__(self, args):
         print(r.text)
 
 
-class YTConfigGetCmd(YTCommand):
+class YTConfigLocalConfigMixin:
```

Comment:
I'm slightly disturbed that this class has "Mixin" in its name but doesn't use inheritance. I don't understand the naming.

Reply:
I changed it to `Handler`.